### ISSUE TITLE
Install instruction doesn't work

### ISSUE THREAD

**Issue by vojty** [barracudanetworks/ArchiveStream-php#21]
`composer install barracudanetworks/archivestream-php`
vs
`composer require barracudanetworks/archivestream-php`

Also version 1.0.0 is missing at packagist https://packagist.org/packages/barracudanetworks/archivestream-php

**Comment by JohnMaguire** [comment]
Thanks @vojty! I'll update the README now. However, you said "version 1.0.0 is missing at packagist". I see it listed there since December 4, 2015. Additionally, using `composer require` I get the following output:

```
-> % composer require 'barracudanetworks/archivestream-php'
Warning: This development build of composer is over 60 days old. It is recommended to update it by running "/usr/local/bin/composer self-update" to get the latest version.
Using version ^1.0 for barracudanetworks/archivestream-php
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing barracudanetworks/archivestream-php (1.0.0)
    Downloading: 100%

Writing lock file
Generating autoload files
```

Can you elaborate on what you mean? Thanks!

**Comment by JohnMaguire** [comment]
Fixed the main issue here with 3eaa040f49c878566768af55e6af98b2e8f6490a.

**Comment by vojty** [comment]
uh, version 1.0.0 is really there, my bad, close then
